generator/types.go

```diff
@@ -368,11 +401,30 @@ func (m *Model) CtorRetVars() string {
 }
 
 // SetFields sets all the children fields and their model to the current model.
+// It also finds the primary key and sets it in the model.
 func (m *Model) SetFields(fields []*Field) {
+	var fs []*Field
+	var id *Field
 	for _, f := range fields {
+		if f.Type == BaseModel {
+			m.ID = f.Fields[0]
+			m.ID.Type = defaultIDType
+			id = f
+		}
+
 		f.Model = m
-		m.Fields = append(m.Fields, f)
+		if f.IsPrimaryKey() {
+			id = f
```

Comment:
let me explain it...
current implementation is:
```
func (m *Model) SetFields(fields []*Field) {
	var id *Field
	for _, f := range fields {
		...
		if f.IsPrimaryKey() {
			id = f
			m.ID = f
		} 
		...
	}
	...
}
```
If the receiver Model has two fields identified as `"kallax:,pk"`, the second one will override the first one.
I think that, in that case (two defined `pk` in the same model), the kallax generation should panic